### DISCUSSION TITLE
[@types/draft-js] not exporting EditorCommand which is needed for handleKeyCommand

### DISCUSSION THREAD

**Discussion by junzhengca** [DefinitelyTyped/DefinitelyTyped#54954] [Issues with a @types package]
The `handleKeyCommand` method has three parameters, the first one being the command sent to the editor. https://draftjs.org/docs/quickstart-rich-styling#richutils-and-key-commands

Apparently it has the type of `EditorCommand`, which is not exported, so as a workaround, I had to do this.

```ts
const handleKeyCommand = (
    command: DraftEditorCommand | string, // This is the type definition for EditorCommand
    editorState: EditorState,
    eventTimeStamp: number
): DraftHandleValue => {
    const newState = RichUtils.handleKeyCommand(editorState, command)
    if (newState) {
        setEditorState(newState)
        return 'handled'
    }
    return 'not-handled'
}
```

which is clearly not future proof, it will break as soon as EditorCommand's type definition changes. So perhaps we should export `EditorCommand`

Reference: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/draft-js/index.d.ts#L199

[a comment by an automated account is not shown]

**Comment by PeterDekkers** [comment]
Yeah, agreed @junzhengca. I'm setting up a PR.

**Comment by PeterDekkers** [comment]
This should do it: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/55958

One of the other maintainers will have to approve the PR.

**Comment by PeterDekkers** [comment]
That PR was merged and should be released soon @junzhengca. Thanks.
